.github/workflows/automated-release.yml

```diff
@@ -72,6 +78,12 @@ jobs:
         run: make build -j8
       - name: Install Github-release
         run: go install github.com/github-release/github-release@latest
+      - name: Configure AWS credentials
```

Comment:
We have configured aws credentials in previous steps. Do we need to reconfigure it again here?